packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,85 @@
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term"
+                    data-translate="xdsl_access_comfort_exchange_rma_reference"></span>
+                <span class="col-sm-4 oui-tile__description" data-ng-bind="rma.id"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term"
+                    data-translate="xdsl_access_comfort_exchange_rma_equipment_reference"></span>
+                <span class="col-sm-4 oui-tile__description" data-ng-bind="rma.equipmentReference"></span>
+            </div>
+        </div>
+    </div>
+
+    <div data-ng-if="!$ctrl.isRMAOpened" class="mb-3">
+        <div data-ng-if="!$ctrl.exchange.isSuccess">
+            <div class="mb-2" data-translate="xdsl_access_comfort_exchange_confirm_title"></div>
+            <div class="mb-2" data-translate="xdsl_access_comfort_exchange_confirm_subtitle"></div>
+
+            <oui-button data-variant="primary" data-ng-disabled="$ctrl.isAvailable"
+                data-ng-click="$ctrl.comfortExchange()">
+                <span data-translate="xdsl_modem_exchange"></span>
+            </oui-button>
+        </div>
+
+        <div data-ng-if="$ctrl.exchange.isSuccess">
+            <h3 data-translate="xdsl_access_comfort_exchange_bc_title"></h3>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term" data-translate="xdsl_access_comfort_exchange_bc_number"></span>
+                <span class="col-sm-4 oui-tile__description" data-ng-bind="$ctrl.exchange.order.orderId"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term" data-translate="xdsl_access_comfort_exchange_bc_date"></span>
+                <span class="col-sm-4 oui-tile__description"
+                    data-ng-bind="($ctrl.exchange.order.date | date: 'dd/MM/yyyy HH:mm:ss' )"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term"
+                    data-translate="xdsl_access_comfort_exchange_price_without_tax"></span>
+                <span class="col-sm-4 oui-tile__description" data-ng-bind="$ctrl.exchange.order.priceWithoutTax.text"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term" data-translate="xdsl_access_comfort_exchange_tax"></span>
+                <span class="col-sm-4 oui-tile__description" data-ng-bind="$ctrl.exchange.order.tax.text"></span>
+            </div>
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term"
+                    data-translate="xdsl_access_comfort_exchange_price_with_tax"></span>
+                <span class="col-sm-4 oui-tile__description" data-ng-bind="$ctrl.exchange.order.priceWithTax.text"></span>
+            </div>
+
+            <div class="row">
```

Comment:
Is this `row` really needed?

Because a `row` always have a `col-x-x` as a child which is not the case here.

ref: https://github.com/twbs/bootlint/wiki/E013

Reply:
If you only have a `col-xx-12` under a `row`, you can safely remove both, this is equivalent to `class="d-block"` :)